src/pages/patientView/copyNumberAlterations/column/AnnotationColumnFormatter.tsx

```diff
@@ -32,13 +41,50 @@ export default class AnnotationColumnFormatter
             value = {
                 myCancerGenomeLinks: [],
                 isHotspot: false,
-                is3dHotspot: false
+                is3dHotspot: false,
+                hasCivicVariants: true
             };
         }
 
         return value;
     }
 
+   /**
+     * Returns an ICivicEntry if the civicGenes and civicVariants have information about the gene and the mutation (variant) specified. Otherwise it returns 
+     * an empty object.
+     */
+    public static getCivicEntry(copyNumberData:DiscreteCopyNumberData[], civicGenes:ICivicGene, civicVariants:ICivicVariant): ICivicEntry | null
+    {
+        let geneSymbol: string = copyNumberData[0].gene.hugoGeneSymbol;
+        let geneVariants: {[name: string]: ICivicVariantData} = civicVariants[geneSymbol];
+        let geneEntry: ICivicGeneData = civicGenes[geneSymbol];
+        let civicEntry = null;
+        //Only return data for genes with variants or it has a description provided by the Civic API
+        if (geneVariants ||
+               (geneEntry && geneEntry.description != "")) {
+            civicEntry = {name: geneEntry.name,
+                             description: geneEntry.description,
+                             url: geneEntry.url,
+                             variants: geneVariants,
+                            }
```

Comment:
A minor thing, but can we fix the indentation here? Something like this would look better:

```
civicEntry = {name: geneEntry.name,
    description: geneEntry.description,
    url: geneEntry.url,
    variants: geneVariants,
};
```

Also, can we add a semicolon at the end of the final line? BTW, it is always useful to run "npm run lint" and review the warnings for the new files/lines introduced. For example, there is a warning for missing semicolons. (You don't need to fix every lint error though)